x/gov/simulation/operations.go

```diff
@@ -34,6 +34,10 @@ const (
 	OpWeightMsgDeposit      = "op_weight_msg_deposit"
 	OpWeightMsgVote         = "op_weight_msg_vote"
 	OpWeightMsgVoteWeighted = "op_weight_msg_weighted_vote"
+
+	DefaultWeightMsgDeposit      = 100 // From simappparams.DefaultWeightMsgDeposit
```

Comment:
I think we can just delete them from `simapp/params` then (as we did for the other, and add a changelog or something)

Reply:
OK, staking should be the last one, so let's not forget to add a changelog entry in that PR. cc @cool-develope